tests/features/group.feature

```diff
@@ -22,6 +22,10 @@ Feature: Subcommand: dab group
 	Scenario: Can group repositories then start them together
 		Given I successfully run `dab repo add three https://github.com/Nekroze/dotfiles.git`
 		And I run `dab repo entrypoint create three start`
+		And I append to "~/.config/dab/repo/three/entrypoint/start" with:
+		"""
+		cat
```

Comment:
won't this cause it to just block until stdin is closed? not sure if stdin is even available during run without piping.